ra/registration-authority.go

```diff
@@ -787,7 +789,40 @@ func (ra *RegistrationAuthorityImpl) UpdateAuthorization(base core.Authorization
 	}
 
 	// Dispatch to the VA for service
-	ra.VA.UpdateValidations(authz, challengeIndex)
+
+	doValidate := func() {
```

Comment:
Why not just move this definition down to where it is invoked?

Perhaps to make the `if` branching easier to read just flip the order (`!ra.useNewVARPC`).